indico/modules/events/templates/display/now_happening.html

```diff
@@ -18,7 +18,7 @@
                 <a href="{{ url }}">{{ obj.title }}</a>
                 {% if obj.room_name %} ({{ obj.room_name }}){% endif %}
                 <em>
-                    {{ entry.start_dt|format_time(timezone=event.timezone) }} - {{ entry.end_dt|format_time(timezone=event.timezone) }}
+                    {{ entry.start_dt|format_time }} - {{ entry.end_dt|format_time }}
```

Comment:
It works fine since when no timezone is provided the function uses `DisplayTZ` to get the correct timezone